src/amp/components/Analytics.tsx

```diff
@@ -11,6 +11,7 @@ export interface AnalyticsModel {
     beacon: string;
     neilsenAPIID: string;
     domain: string;
+    permutive: { [key: string]: any };
```

Comment:
Given we (I presume) know the structure of this object

```typescript
            namespace: 'guardian',
            apiKey: '42-2020',
            payload: {
                headline: 'article title',
            },
```
Would it be possible to drop the `any` here and define a stricter type?

I'm probably getting lazy but I found it difficult to reason through the code without types to guide me!